mne/simulation/raw.py

```diff
@@ -271,8 +276,10 @@ def simulate_raw(info, stc=None, trans=None, src=None, bem=None, head_pos=None,
     logger.info('Setting up raw simulation: %s position%s, "%s" interpolation'
                 % (len(dev_head_ts), _pl(dev_head_ts), interp))
 
+    if isinstance(stc, SourceSimulator) and stc.first_samp != first_samp:
+        logger.info('SourceSimulator first_samp does not match argument.')
```

Comment:
Shouldn't we warn here? Or is it really so "unimportant"?

Reply:
I'm not sure of the convention MNE has for generating a warning.  There could be valid reasons that a user would originally add events to SourceSimulator using one first_samp value but then declare a different first_samp value in simulate_raw();  e.g. to align the raw data with another raw or reference waveform.  Still figured some sort of indicator in the log was warranted.